README.md

```diff
@@ -206,13 +206,13 @@ The `stable` key refers to hosts available for the stable release of that versio
 to all pre-releases and dev snapshots, which typically share all their characteristics. If in future there is a
 need for finer control, some overrides system needs to be implemented.
 
-For the logic side of things the new host needs to be supported by the `_plugins/make_download.rb` script. Refer
+For the logic side of things, the new host needs to be supported by the `_plugins/make_download.rb` script. Refer
 to how other hosts are handled in that file and do the necessary adjustments. We assume that the final filenames
 are standard across all hosts, so `_data/download_configs.yml` is respected.
 
 ## Resources
 
 - Join the discussion on Godot Contributors Chat in the [#website](https://chat.godotengine.org/channel/website)
   channel.
-- Please, consider the [website usage stats](https://stats.tuxfamily.org/godotengine.org) when relying on modern
+- Please consider [website usage stats](https://stats.tuxfamily.org/godotengine.org) when relying on modern
```

Comment:
Since we're no longer hosting the website on TuxFamily, I'm not sure how relevant this still is.

Reply:
I think that line can be removed entirely, or we can point to https://caniuse.com/.

Reply:
Seconding caniuse as a de facto standard.

Reply:
Yeah that makes sense, I missed the URL it links to - changed to caniuse.com.